src/test/resources/META-INF/mods.toml

```diff
@@ -243,6 +243,8 @@ modId="mega_model_test"
 [[mods]]
 modId="emissive_elements_test"
 [[mods]]
+modId="anvil_update_event_fix"
+[[mods]]
 modId="item_use_animation_test"
```

Comment:
This is weird, it looks fine on original file https://github.com/tmvkrpxl0/MinecraftForge/blob/itemanim-1.19.x/src/test/resources/META-INF/mods.toml
But not in here